nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -169,6 +169,19 @@ in {
       default = [];
     };
 
+    strip = mkOption {
```

Comment:
This doesn't appear to be getting passed to the `makeInitrdNG`, does it?

Reply:
Looks like it. ping @lheckemann

Reply:
whooops yes, accidentally left that unstaged.

Reply:
https://github.com/lheckemann/nixpkgs/commit/c9b29fccbf5b132a7b0f7b61a7404622482d3b64

Reply:
Picked

Reply:
Why do we need an option for this at all? When would I want to turn this off?

Reply:
As the description says, it's a pretty small initrd size increase for a much bigger build closure decrease, so some people might want to take that trade.